vision_datasets/common/data_manifest.py

```diff
@@ -96,14 +98,28 @@ def __init__(self, id, img_path, width, height, labels):
                 classification: [c_id] for multiclass, [c_id1, c_id2, ...] for multilabel;
                 detection: [[c_id, left, top, right, bottom], ...];
                 image caption: [caption1, caption2, ...];
-                image_text_matching: [(text1, match (0 or 1), text2, match (0 or 1), ...)]
+                image_text_matching: [(text1, match (0 or 1), text2, match (0 or 1), ...)];
+                image matting: [image_matting], where 'image_matting' is a PIL object or a Numpy array;
                 multitask: dict[task, labels]
+            label_file_path (str): path to label file
```

Comment:
Let us make the comment clear that this label_file_path only works for matting for now